docs/source/guide/prompts_overview.md

```diff
@@ -135,7 +135,7 @@ This feedback loop allows you to iteratively fine-tune your prompts, optimizing
 | **Supported data types** | Text |
 | **Supported model types** | Text Classification <br>Named Entity Recognition (NER) |
 | **Class selection** | Multi-selection (the LLM can apply multiple labels per task)|
-| **Supported base models** | OpenAI gpt-3.5-turbo-16k <br>OpenAI gpt-3.5-turbo-instruct* <br>OpenAI gpt-4-turbo <br>OpenAI gpt-3.5-turbo <br>OpenAI gpt-4o <br>OpenAI gpt-4<br>[Azure OpenAI chat-based models](https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/models) |
+| **Supported base models** | OpenAI gpt-3.5-turbo-16k <br>OpenAI gpt-3.5-turbo-instruct* <br>OpenAI gpt-4-turbo <br>OpenAI gpt-3.5-turbo <br>OpenAI gpt-4o <br>OpenAI gpt-4<br>[Azure OpenAI chat-based models](https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/models)<br>[Custom LLM](prompts_create#Model-provider-API-keys) |
```

Comment:
is `prompts_create#Model-provider-API-keys` a right link? is it possible to pin the exact section?

Reply:
Because it's in an expandable section without a heading, I can't link directly to it. But I did add a new heading that will take them closer to it.